Library/Formula/sphinx.rb

```diff
@@ -21,13 +22,18 @@ class Sphinx < Formula
     sha256 "a71c292f774b8e449cf8775d21e7a6f41a85497e178ea6ba678b80a2f33b4ebd" => :mountain_lion
   end
 
-  option 'mysql', 'Force compiling against MySQL'
-  option 'pgsql', 'Force compiling against PostgreSQL'
-  option 'id64',  'Force compiling with 64-bit ID support'
+  option 'with-mysql',      'Force compiling against MySQL'
+  option 'with-postgresql', 'Force compiling against PostgreSQL'
+  option 'with-id64',       'Force compiling with 64-bit ID support'
+
+  deprecated_option 'mysql' => 'with-mysql'
+  deprecated_option 'pgsql' => 'with-postgresql'
+  deprecated_option 'id64'  => 'with-id64'
 
   depends_on "re2" => :optional
-  depends_on :mysql if build.include? 'mysql'
-  depends_on :postgresql if build.include? 'pgsql'
+  depends_on :mysql => :optional
+  depends_on :postgresql => :optional
+  depends_on 'openssl' if build.with?('mysql')
```

Comment:
It doesn't linking with OpenSSL when compiled with PostgreSQL support, so no need in OpenSSL there.